RemoveNthFromEnd.java

```diff
@@ -0,0 +1,20 @@
+public class RemoveNthFromEnd {
+    //Time Complexity = O(1)
+    //Space Complexity = O(1)
+    public ListNode reverseList(ListNode head) {
+        if(head==null || head.next==null)
```

Comment:
Also this is removing nth node from end . You have submitted solution for a different problem